problems/prob2.rb

```diff
@@ -0,0 +1,15 @@
+#Given an integer array of size N, identify the largest 2 numbers in this array.
+#Do not sort the array.
+
+def largest_two_numbers(numbers)
+  if numbers.nil? || numbers.empty? || numbers.any? { |e| e == nil  }
```

Comment:
This is not needed, but if it was needed is considered a code smell to repeat the same snippet of code.

Now that you have this around can you [dry'it](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Goes to prove that if you write more code than needed it will generate even more workload :)

Reply:
Constantin suggested that I should not check for nil given the problem description and I removed some parts in the latest commit.

Reply:
Indeed that is better,

I was just making a point that more code, even if it's necessarily good code, will open the way for issues.